src/main/java/com/epam/ta/reportportal/dao/LaunchRepository.java

```diff
@@ -74,4 +75,14 @@
 			+ " WHERE ti.launch_id = :launchId AND tir.status <> cast(:#{#status.name()} as status_enum) LIMIT 1)", nativeQuery = true)
 	boolean hasItemsWithStatusNotEqual(@Param("launchId") Long launchId, @Param("status") StatusEnum status);
 
+	/**
+	 * Finds the latest(that has max {@link Launch#number) {@link Launch} with specified {@code name} and {@code projectId}
+	 *
+	 * @param name      Name of {@link Launch}
+	 * @param projectId Id of {@link Project}
+	 * @return {@link Optional<Launch>} if exists, {@link Optional#empty()} if not
+	 */
+	@Query(value = "select * from launch l where l.name =:name and l.project_id=:projectId order by l.number desc limit 1", nativeQuery = true)
```

Comment:
formatting of native queries should be in upper case by default

Reply:
fixed